nucliadb_dataset/nucliadb_dataset/run.py

```diff
@@ -20,24 +20,25 @@
 from pathlib import Path
 from urllib.parse import urlparse
 
-import pydantic_argparse
+import argdantic
 from nucliadb_protos.dataset_pb2 import TrainSet
 
 from nucliadb_dataset import ExportType
 from nucliadb_dataset.dataset import TASK_DEFINITIONS
 from nucliadb_dataset.export import FileSystemExport, NucliaDatasetsExport
+from nucliadb_dataset.settings import RunningSettings
 from nucliadb_sdk.v2.sdk import NucliaDB
 
+parser = argdantic.ArgParser()
 
-def run():
-    from nucliadb_dataset.settings import RunningSettings
 
-    parser = pydantic_argparse.ArgumentParser(
-        model=RunningSettings,
-        prog="NucliaDB Datasets",
-        description="Generate Arrow files from NucliaDB KBs",
```

Comment:
nit: can you migrate the descriptions/prog?

Reply:
I tried with the program name and it doesn't change anything in argdantic output. The description does, so I'll copy that.